pkg/interface/src/themes/light.ts

```diff
@@ -162,7 +162,7 @@ const theme = {
   ],
   // z-index
   zIndices: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
-  breakpoints: ["768px", "1024px", "1440px", "2200px"],
+  breakpoints: ["34.375em", "46.875em", "60em"]
```

Comment:
yes

Reply:
hang on, why did we go with ems here?

Reply:
to match our current breakpoints in indigo-static

Reply:
Actually @liam-fitzgerald brought up out of band that there is no way to do anything responsively with a minimum of 768 also, this seems like a valid concern.

Reply:
Ok, I need to check on this because it seems counter to the [styled-system documentation](https://styled-system.com/responsive-styles), unless I'm misunderstanding these docs. I'd also prefer we use px for breakpoints. You can just multiply ems by 16 to get their px values.

Reply:
Not exactly. Really it should be rems but at this point there won't be any divergence. I always use rems because that way the interface scales in proportion to whatever the user sets their font size at. But I don't know what kind of guidelines this project follows. I've actually transitioned to setting font-size in millimeters at root and everything else to rem which bases it off that. That grounds it in physical reality.